streamlit_webrtc/process.py

```diff
@@ -134,7 +134,10 @@ async def _fallback_recv_queued(self, frames: List[FrameT]) -> FrameT:
                 "Some frames have been dropped. "
                 "`recv_queued` is recommended to use instead."
             )
-        return [self.processor.recv(frames[-1])]
+        if self.processor.recv:
```

Comment:
Should `self.processor` be renamed to something, e.g. `.callbacks`?

Reply:
Don't.

Due to https://github.com/whitphx/streamlit-webrtc/pull/747#discussion_r889661893